apps/src/templates/rubrics/LearningGoals.jsx

```diff
@@ -0,0 +1,392 @@
+                  }
+                </BodyThreeText>
+              )}
+              {submittedEvaluation.feedback && (
+                <FontAwesome
+                  icon="message"
+                  className="fa-regular"
+                  title={i18n.feedback()}
+                />
+              )}
+            </div>
+          )}
+        </div>
+        <button
+          type="button"
+          className={style.learningGoalButton}
+          onClick={() => onCarouselPress(1)}
+        >
+          <FontAwesome icon="angle-right" />
+        </button>
+      </div>
+
+      {/*TODO: Pass through data to child component*/}
+      <div>
+        {teacherHasEnabledAi &&
+          !!studentLevelInfo &&
+          !!aiEvalInfo &&
+          aiUnderstanding !== undefined && (
+            <div className={style.openedAiAssessment}>
+              <AiAssessment
+                isAiAssessed={learningGoals[currentLearningGoal].aiEnabled}
+                studentName={studentLevelInfo.name}
+                aiConfidence={aiConfidence}
+                aiUnderstandingLevel={aiUnderstanding}
+                aiEvalInfo={aiEvalInfo}
+              />
+            </div>
+          )}
+        <div className={style.learningGoalExpanded}>
+          {!!submittedEvaluation && renderSubmittedFeedbackTextbox()}
+          <EvidenceLevels
+            learningGoalKey={learningGoals[currentLearningGoal].key}
+            evidenceLevels={learningGoals[currentLearningGoal].evidenceLevels}
+            canProvideFeedback={canProvideFeedback}
+            understanding={displayUnderstanding}
+            radioButtonCallback={radioButtonCallback}
+            submittedEvaluation={submittedEvaluation}
+            isStudent={isStudent}
+          />
+          {learningGoals[currentLearningGoal].tips && !isStudent && (
+            <div>
+              <Heading6>{i18n.tipsForEvaluation()}</Heading6>
+              <div className={style.learningGoalTips}>
+                <SafeMarkdown
+                  markdown={learningGoals[currentLearningGoal].tips}
+                />
+              </div>
+            </div>
+          )}
+          {!!studentLevelInfo && renderAutoSaveTextbox()}
+        </div>
+      </div>
+    </div>
+  );
+}
+
+LearningGoals.propTypes = {
+  teacherHasEnabledAi: PropTypes.bool,
+  canProvideFeedback: PropTypes.bool,
+  reportingData: reportingDataShape,
+  studentLevelInfo: studentLevelInfoShape,
+  submittedEvaluation: submittedEvaluationShape,
+  isStudent: PropTypes.bool,
+  feedbackAdded: PropTypes.bool,
+  setFeedbackAdded: PropTypes.func,
+  learningGoals: PropTypes.arrayOf(learningGoalShape),
+  aiEvaluations: PropTypes.arrayOf(aiEvaluationShape),
+};
+
+const AiToken = () => {
+  return (
+    <div className="uitest-uses-ai">
+      {' '}
+      <BodyFourText className={classnames(style.aiToken, style.aiTokenText)}>
+        <ExtraStrongText>{i18n.usesAi()}</ExtraStrongText>
+      </BodyFourText>
+    </div>
+  );
+};
```

Comment:
Tiny nit: when I'm writing a function that only has one statement returning a value I prefer writing it like this because it saves two lines and one level of indentation.
```suggestion
const AiToken = () => (
  <div className="uitest-uses-ai">
    {' '}
    <BodyFourText className={classnames(style.aiToken, style.aiTokenText)}>
      <ExtraStrongText>{i18n.usesAi()}</ExtraStrongText>
    </BodyFourText>
  </div>
);
```

Reply:
I've got failing tests looking for a return statement on this function when I try it this way.

Reply:
Huh, interesting. I wonder why. A new lint rule, perhaps? Leave it the way it was I guess.